src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -87,9 +102,9 @@ public boolean equals(Object other) {
                 return false;
             }
             DeleteCommand commandToCompare = (DeleteCommand) other;
-            if (this.targetId == null && this.targetIndex != null) { // only targetIndex present
-                return targetIndex.equals(commandToCompare.targetIndex); // state check
-            } else if (this.targetIndex == null && this.targetId != null) { // only targetId present
+            if (this.targetId == null && this.targetIndices != null) { // only targetIndex present
```

Comment:
```suggestion
            if (this.targetId == null && this.targetIndices != null) { // only targetIndices present
```

I think this might be a typo.

Reply:
Thanks for the catch!